force-app/main/default/lwc/courseRegistrationInformation/courseRegistrationInformation.html

```diff
@@ -0,0 +1,29 @@
+<template>
+    <lightning-record-view-form record-id={courseId} object-api-name="Course__c">
+
+        <div class="slds-box slds-theme_shade">
+
+            <div class="slds-text-heading_medium">Informasjon</div>
+
+            <p if:false={courseId}> Vennligst gå tilbake til kursoversikten og velg riktig kurs:
+                https://arbeidsgiver.nav.no/kursoversikt/ </p>
+
+            <div class="slds-text-title_bold">Sted</div>
```

Comment:
```suggestion
            <div class="slds-text-title_bold slds-p-top_medium">Sted</div>
```

Legger til medium padding over 'Sted'